bootstrap.sh

```diff
@@ -1,16 +1,29 @@
 #!/bin/bash
 
-export PATH=./cmd/authelia-scripts/:/tmp:$PATH:./node_modules/.bin
+export PATH=$PATH:./cmd/authelia-scripts/:./node_modules/.bin:/tmp
 
 if [ -z "$OLD_PS1" ]; then
   OLD_PS1="$PS1"
   export PS1="(authelia) $PS1"
 fi
 
-export USER_ID=$(id -u)
-export GROUP_ID=$(id -g)
-export CI=false
+if [ $(id -u) = 0 ]; then
+  export USER_ID=1000
```

Comment:
Put a warning log here saying that the uid of the user cannot be root and defaults to 1000.